recipes/eliot/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "eliot" %}
+{% set version = "1.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 973e5c9518b24a85d4ed2841882e32359ab5cdaf8389fc98dbd24866f052b84c
+
+build:
+  number: 0
+  skip: true  # [py<35]
```

Comment:
If you remove this selector and turn it into a python version constraint, I think you make this package noarch: python. See the conda-forge documentation here: https://conda-forge.org/docs/meta.html#building-noarch-packages

Reply:
Thanks, will check it out.

Reply:
OK, done.